### PR TITLE
Infeed video ad

### PR DESCRIPTION
**Pull request by anshumansworld**
This includes implementation of feed video in library. For this a new module `library.feed.video` has been added.

### CODE REVIEW

File: library.feed.video/src/main/java/net/pubnative/library/feed/video/PubnativeFeedVideo.java [hunk 1]
```diff
@@ -0,0 +1,460 @@
...
+        if (mIsLoading) {
+            Log.w(TAG, "show - the ad is still loading, dropping the call");
+        } else if (mIsShown) {
+            Log.w(TAG, "show - the ad is already shown, ");
+        } else if (isReady()) {
+            mContainer = container;
+            render();
+        } else {
+            Log.e(TAG, "show - Error: the video is not yet loaded");
+        }
```
Comment (cerberillo) [inline library.feed.video/src/main/java/net/pubnative/library/feed/video/PubnativeFeedVideo.java]: What if we also check about the ocntainer nullity? :)

File: library.feed.video/src/main/java/net/pubnative/library/feed/video/PubnativeFeedVideo.java [hunk 2]
```diff
@@ -0,0 +1,460 @@
...
+
+                    @Override
+                    public void onPrepareLoad(Drawable placeHolderDrawable) {
+
+                    }
+                });
+                mVASTPlayer.load(model);
+                RelativeLayout.LayoutParams params = new RelativeLayout.LayoutParams(SystemUtils.dpToPx(mContext, 320), SystemUtils.dpToPx(mContext, 250));
+                params.addRule(RelativeLayout.CENTER_IN_PARENT);
+                mContainer.addView(mFeedVideo, params);
```
Comment (cerberillo) [inline library.feed.video/src/main/java/net/pubnative/library/feed/video/PubnativeFeedVideo.java]: Please do all the load once the picasso banner is loaded, not before.

File: library.demo/src/main/AndroidManifest.xml [hunk 3]
```diff
@@ -24,8 +23,10 @@
         <activity android:name=".InterstitialAdActivity" />
         <activity android:name=".BannerActivity" />
         <activity android:name=".InFeedBannerActivity" />
-        <activity android:name=".VideoAdActivity"
-            android:configChanges="orientation"/>
+        <activity
+            android:name=".VideoAdActivity"
+            android:configChanges="orientation" />
```
Comment (cerberillo) [inline library.demo/src/main/AndroidManifest.xml]: I thought we talked we were going to use WindowManager instead of VideoAdActivity for both video banners. Is this still valid? In which ase, why is this here?

File: library.feed.video/src/main/java/net/pubnative/library/feed/video/PubnativeFeedVideo.java (hunk 2 above)
Comment (anshumansworld) [inline library.feed.video/src/main/java/net/pubnative/library/feed/video/PubnativeFeedVideo.java]: this is post banner so I guess its fine to load the player and in the background banner would be loaded. Moreover this banner is already fetched in `onPubnativeRequestSuccess`.

File: library.demo/src/main/AndroidManifest.xml (hunk 3 above)
Comment (anshumansworld) [inline library.demo/src/main/AndroidManifest.xml]: yes we are doing through WindowManager. This is for demo app in which we are opening an activity on selection of different type of ads and from there we make a request.

Comment (cerberillo) [inline library.demo/src/main/AndroidManifest.xml]: what I mean is what is the VideoAdActivity for if we're using windowmanager? is it required?

**Comment by cerberillo** [comment]
This is ready to merge 👍
